kolibri/core/utils/lock.py

```diff
@@ -73,3 +74,27 @@ def db_lock():
                 vendor=connection.vendor
             )
         )
+
+
+def retry_on_db_lock(func, retries=5):
```

Comment:
I don't think there's a way to change the value of this kwarg when this is used as a decorator, right?

Not important, but could just set it as a local variable instead. I forget the proper way to do optional decorator arguments.

Reply:
Yeah you would have to double wrap it I think, but since I wasn't changing it, I didn't bother implementing. If you think this is confusing I can make it local

Reply:
Might avoid confusing a future user of the function.